notebooks/useful_notebooks/01_find_and_release.ipynb

```diff
@@ -212,15 +215,45 @@
     "            msg = ('MATCHING ACCESS STATUS', a_type, raw_data['uuid'], raw_data['status'])\n",
     "        elif STATUS_LEVEL.get(raw_data['status']) == 0:\n",
     "            unusual += 1\n",
-    "            msg = ('SKIP UNUSUAL STATUS  ', a_type, raw_data['uuid'], raw_data['status'])\n",
+    "            msg = ('SKIP UNUSUAL STATUS   ', a_type, raw_data['uuid'], raw_data['status'])\n",
     "        else:\n",
     "            change += 1\n",
-    "            msg = ('       CHANGE        ', a_type, raw_data['uuid'], raw_data['status'])\n",
+    "            msg = ('        CHANGE        ', a_type, raw_data['uuid'], raw_data['status'])\n",
```

Comment:
Is this block only on the release part? The cell before this one is a preview of changes, will we see a message about files to be restricted there? 
It would be nice if this was a function in dcic repo, so the notebook does not get longer, and can be used in multiple places. You can check the get_key function in dcicutils to see a simple function.